RNPComponents/RNPComponents.md

```diff
@@ -0,0 +1,476 @@
+    >
+        Label
+    </Button>
+    <Button mode="contained" disabled >
+        Label
+    </Button>
+    ```
+
+  - #### Outlined Mode
+
+    Recommended style overrides:
+    * text color of the error outlined button: error
+
+    <img width='300' alt="Button Outlined Mode" src="./Images/buttonOutline.png"><br/>
+
+    Usage
+    ```tsx
+    import { Button } from 'react-native-paper';
+
+    <Button mode="outlined">
+        Label
+    </Button>
+    <Button
+        mode="outlined"
+        textColor={theme.colors.error}
+    >
+        Label
+    </Button>
+    <Button mode="outlined" disabled>
+        Label
+    </Button>
+    ```
+
+  - #### Text Mode
+
+    Recommended style overrides:
+    * text color of the error text button: error
+
+    <img width='300' alt="Button Text Mode" src="./Images/buttonText.png"><br/>
+
+    Usage
+    ```tsx
+    import { Button } from 'react-native-paper';
+
+    <Button mode="text">
+        Label
+    </Button>
+    <Button
+        mode="text"
+        textColor={theme.colors.error}
+    >
+        Label
+    </Button>
+    <Button mode="text" disabled>
+        Label
+    </Button>
+    ```
+  - #### Elevated Mode
+
+    Recommended style overrides:
+    * button color of the elevated button: surfaceContainerLow
+    * text color of the error elevated button: error
+    * button color of the error elevated button: surfaceContainerLow
+    * button color of the disabled elevated button: surfaceContainerLow
+
+    <img width='300' alt="Button Elevated Mode" src="./Images/buttonElevated.png"><br/>
+
+    Usage
+    ```tsx
+    import { Button } from 'react-native-paper';
+
+    <Button
+        mode="elevated"
+        buttonColor={theme.colors.surfaceContainerLow}
+    >
+        Label
+    </Button>
+    <Button
+        mode="elevated"
+        textColor={theme.colors.error}
+        buttonColor={theme.colors.surfaceContainerLow}
+    >
+        Label
+    </Button>
+    <Button
+        mode="elevated"
+        disabled
+        buttonColor={theme.colors.surfaceContainerLow}
+    >
+        Label
+    </Button>
+    ```
+
+  - #### Contained Tonal Mode
+
+    Recommended style overrides:
+    * text color of the error text button: onErrorContainer
+    * button color of the error text button: errorContainer
+
+    <img width='300' alt="Button Contained-tonal Mode" src="./Images/buttonTonal.png"><br/>
+
+    Usage
+    ```tsx
+    import { Button } from 'react-native-paper';
+
+    <Button mode="contained-tonal">
+        Label
+    </Button>
+    <Button
+        mode="contained-tonal"
+        buttonColor={theme.colors.errorContainer}
+        textColor={theme.colors.onErrorContainer}
+    >
+        Label
+    </Button>
+    <Button mode="contained-tonal" disabled>
+        Label
+    </Button>
+    ```
+
+- ###  Data Table
+
+    Recommended style overrides:
+    * Text color of Table Title: onSurface
+    * font size of Table Title: theme.fonts.labelLarge
```

Comment:
Be consistent with your letter casing. I see a mix of uppercase and lowercase letters used arbitrarily

Reply:
(many other paragraphs have a similar problem; I will not repeat)